packages/svelte-ux/src/lib/components/DateRangeField.svelte

```diff
@@ -104,7 +103,7 @@
     on:click={() => (open = true)}
     {id}
   >
-    <DateRangeDisplay {value} {format} />
+    <DateRangeDisplay {value} />
   </button>
```

Comment:
I put this here, to track it. I'll look later (there are maybe other places). It's cool to check it out with prod doc & PR doc 🎉

> The range now shows numeric instead of month abbreviations (`12/29/2023` instead of `Dec 29, 2023`).  It also shows leading `0`s for months/days (`01/01/2024` instead of `1/1/2024`).  Neither are major, and I'm sure we can change/tweak the default for some of these cases.  I also need to think if it's better or worse than before :).

Before | After
--- | ---
![image](https://github.com/techniq/svelte-ux/assets/177476/803ca36b-7efe-40a8-a7d7-70664105a3e3) | ![image](https://github.com/techniq/svelte-ux/assets/177476/d38c5efd-83da-481c-9296-c8f6bb20511d)

Reply:
Hooo, yes, now I remember ^^.
Because in the `days.default`, I've set by default default `baseParsing` that is: 'MM/dd/yyyy'.
I can remove this, behavior, be we need to test that parsing is still working good.

I'll revert this, we will see if parsing will be an issue.